### PR TITLE
Niva requested content

### PR DESCRIPTION
**Pull request by vivigirardin**

### CODE REVIEW

File: doc/specialized_userportals_template.adoc [hunk 1]
```diff
@@ -1,26 +1,30 @@
...
-|
-|
-|
+|AquaMonitor
+|Water monitoring portal for access of chemical and biological data.
+|General?
+|Internals and externals
+|Logger's data platform
+|Quality control and visualization of time series data from stationary loggers
+|Targeted?
```
Comment (IvanaH8) [inline doc/specialized_userportals_template.adoc:23]: Here I think it is targeted

File: docs/index.html [hunk 2]
```diff
@@ -662,7 +551,7 @@ <h2 id="_revision_history" class="discrete">Revision history</h2>
 <h2 id="introduction"><a class="anchor" href="#introduction"></a>1. Introduction</h2>
 <div class="sectionbody">
 <div class="paragraph">
-<p>The purpose of the Data Management Handbook (<a href="#dmh">DMH</a>) is threefold:</p>
+<p>The purpose of NIVA&#8217;s Data Management Handbook (<a href="#dmh">DMH</a>) is threefold:</p>
```
Comment (IvanaH8) [inline docs/index.html:554]:
```suggestion
<p>The purpose of NIVA's Data Management Handbook (<a href="#dmh">DMH</a>) is threefold:</p>
```

**Review by IvanaH8 — APPROVED** [review]
Nice changes and additions! Only a couple of suggestions

File: doc/specialized_intro_template.adoc [hunk 3]
```diff
@@ -23,9 +27,20 @@
...
+    - Development of document systems
+
+   
+    - Data quality assurance 
+|Data engineer
+|    Data engineer ensures that data are collected and correctly uploaded to NIVA's data storages
+|    
+     -   Reporting of data to https://vannmiljo.miljodirektoratet.no[Vannmiljø] 
+
+     - Quality assurance
```
Comment (IvanaH8) [inline doc/specialized_intro_template.adoc:46]:
```suggestion
|===
```
To end the table

File: docs/index.html (hunk 2 above)
Comment (vivigirardin) [reply docs/index.html:554]: Not sure why it was written in unicode... I certainly didn't write it like that!

Comment (IvanaH8) [reply docs/index.html:554]: Probably some auto-update glitch...

File: doc/introduction.adoc [hunk 4]
```diff
@@ -4,21 +4,17 @@
...
+The purpose of NIVA's Data Management Handbook (<<dmh,DMH>>) is threefold:
 
 1. to provide an overview of the principles for data management to be employed;
 2. to help personnel identify their roles and responsibilities for good data management; and
 3. to provide personnel with practical guidelines for carrying out good data management.
 
-<<glossary-data-management,_Data management_>> is the term used to describe the handling of data in a systematic and cost-effective manner. The data management regime should be continuously evolving, to reflect the evolving nature of data collection. Therefore this <<dmh,DMH>> is a living document that will be revised and updated from time to time in order to maintain its relevance.
+This document covers the following topics:
 
-The <<dmh,DMH>> is a strategic governing document and should be used as part of the quality framework the organisation is using.
```
[the remark this answers is not in the document]
Comment (vivigirardin) [reply doc/introduction.adoc:15]: Yes. We actually only need to files starting with _specialized_...

https://github.com/NIVANorge/data-management-handbook/blob/f6d4aadac755c827f9c13280b196ce8733aab8a0/README.md?plain=1#L19-L28

**Pull request merged**